samples/sdl_gamecontroller/Makefile

```diff
@@ -1,7 +1,7 @@
 XBE_TITLE = nxdk\ sample\ -\ SDL2_GameController
 GEN_XISO = $(XBE_TITLE).iso
 SRCS = $(CURDIR)/main.c
-NXDK_DIR = $(CURDIR)/../..
+NXDK_DIR ?= $(CURDIR)/../..
 NXDK_SDL = y
 
```

Comment:
Might want to delete this empty line in order to unify this Makefile with the majority.